app/init.php

```diff
@@ -3,9 +3,17 @@
 $dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
 $dotenv->load();
 
+$loader = new \Twig\Loader\FilesystemLoader(__DIR__ . '/templates');
+$twigConfig = [];
+if (getenv('PRODUCTION')) {
```

Comment:
Next, we check if this is a production server, and if it is, we add a `cache` setting to the `$twigConfig` array. For now, this is the only twig configuration setting.